cypher/cypher-docs/src/test/java/org/neo4j/cypher/export/SubGraphExporter.java

```diff
@@ -87,32 +88,48 @@ private void output( PrintWriter out, String ... commands )
         {
             if ( !index.isConstraintIndex() )
             {
-                Iterator<String> propertyKeys = index.getPropertyKeys().iterator();
-                if ( !propertyKeys.hasNext() )
+                if ( index.getIndexType().equals( IndexType.LOOKUP ) )
```

Comment:
I'm not really sure what this SubGraphExporter is used for/where it is rendered so a bit hard to review...

Reply:
it is used for tests with multiple test queries, it helps exporting constraints/indexes between the different queries. This is the part which allows all single property test queries to be in one test but forces all composite test queries to have individual tests (since it does not handle multiple properties). I guess I could leave this and just have 2 lookup tests to avoid the issue, but it looked like it needed an upgrade anyway

was missing exporting names, so if you had
```
test("..."){
testQuery("CREATE INDEX foo FOR ...")
testQuery("SHOW INDEXES")
}
```
you would not have had the name `foo` but a new generated one in the second test query output

I only know that it does because I keep falling into cases it can't export and have to split tests :P